### ISSUE TITLE
Release/tag v0.1.2?

### ISSUE THREAD

**Issue by DavidLemayian** [okfn/ckanext-s3filestore#27]
A few changes to fix mime-type might be good to tag a new patch release? https://github.com/okfn/ckanext-s3filestore/compare/v0.1.1...master

**Comment by adborden** [comment]
Hello, we'd love to see an official tag published for the mimetype fix.

**Comment by mbocevski** [comment]
@adborden we have that fixed + chrome issues + ckan 2.9 support on our fork here: https://github.com/keitaroinc/ckanext-s3filestore. We will submit them upstream as well however we don't see a lot of maintenance happening here.

**Comment by adborden** [comment]
Thanks @mbocevski ! I know [qld.gov.au is also keeping an up to date fork](https://github.com/qld-gov-au/ckanext-s3filestore). Can OKFN mark this repo as archived and link to a new upstream? Maybe publish on [pypi](https://pypi.org/project/ckanext-s3filestore/)? It's hard to know which forks are actively maintained in the CKAN community.

**Comment by ThrawnCA** [comment]
@adborden Yes, we are indeed maintaining the qld-gov-au fork. I'm interested in the mention of CKAN 2.9 support; we'll probably migrate early next year if all goes well, so I'll probably take a look at what keitaroinc has done.

**Comment by ThrawnCA** [comment]
@mbocevski I can't see an option to raise issues on your fork, but you might want to take a look at ours and see if there's any features you want to incorporate.

- Support for using AMI roles instead of hardcoding an access key.
- Bucket is not checked/created until it's actually used, so command-line invocations (eg paster) have less overhead.
- Support for substituting the hostname on signed URLs so it's easier to use CloudFront.
- Better API compatibility with core, so it's easier to interact with other extensions like ckanext-archiver.
- Specifying mime types on uploads.
- Support for deleting files from S3, or obtaining metadata.
- Automatic detection of file size.

I'm guessing that the main CKAN 2.9 change is the blueprint-related code?

**Comment by mbocevski** [comment]
@ThrawnCA Thanks, we probably have turned off issues at an earlier stage, I've enabled issues on our fork now. We will definitely take a look at your fork and incorporate your changes and features. I've created https://github.com/keitaroinc/ckanext-s3filestore/issues/12 to track this.

Yes mainly the blueprint code is related to ckan 2.9

**Comment by amercader** [comment]
@ThrawnCA @mbocevski @adborden 
I've marked this repo as no longer maintained and directed users to your two forks in the README. It would be great to consolidate efforts in a single repo, which could be hosted in the ckan organization if that was desirable.

I haven't archived the repo yet as it would prevent new comments on issues but will do in a few days.

**Comment by mbocevski** [comment]
Thanks @amercader. @ThrawnCA @adborden please let me know how you want to continue forward, we have no issues of integrating functionalities from the community and other forks in our fork and keep on maintaining it going forward.

**Comment by ThrawnCA** [comment]
@mbocevski We would be interested in merging repositories to a point where we can install a version from PyPI that has everything we want, instead of from Github. We don't necessarily want ownership of the extension, just a good turnaround time when we want to fix bugs or add features; maintaining a fork has worked out thus far, but we're not married to the idea. How about yourselves?

**Comment by mbocevski** [comment]
@ThrawnCA I think that's a good approach. Let's then merge and get releases out to pypi, we can maintain the fork cause we are maintaining for wide use-case scenarios. I think we can collaborate well, let's try out and see how it goes.
